Feliz/Styles.fs

```diff
@@ -89,410 +94,652 @@ type style =
+    /// Sets the minimum height of an element. 
+    ///
+    /// It prevents the used value of the height property from becoming smaller than the value specified for min-height.
     static member inline minHeight (value: int) = Interop.mkStyle "minHeight" value
+    /// Sets the minimum height of an element. 
+    ///
+    /// It prevents the used value of the height property from becoming smaller than the value specified for min-height.
     static member inline minHeight (value: ICssUnit) = Interop.mkStyle "minHeight" value
+    /// Sets the maximum width of an element. 
+    ///
+    /// It prevents the used value of the width property from becoming larger than the value specified by max-width.
     static member inline maxWidth (value: int) = Interop.mkStyle "maxWidth" value
+    /// Sets the maximum width of an element. 
+    ///
+    /// It prevents the used value of the width property from becoming larger than the value specified by max-width.
     static member inline maxWidth (value: ICssUnit) = Interop.mkStyle "maxWidth" value
+    /// Sets the maximum height of an element. 
+    ///
+    /// It prevents the used value of the height property from becoming larger than the value specified for max-height.
     static member inline maxHeight (value: int) = Interop.mkStyle "maxHeight" value
+    /// Sets the maximum height of an element. 
+    ///
+    /// It prevents the used value of the height property from becoming larger than the value specified for max-height.
     static member inline maxHeight (value: ICssUnit) = Interop.mkStyle "maxHeight" value
+    /// Set the height of an element. 
+    ///
+    /// By default, the property defines the height of the content area.
     static member inline height (value: int) = Interop.mkStyle "height" value
+    /// Set the height of an element. 
+    ///
+    /// By default, the property defines the height of the content area.
     static member inline height (value: ICssUnit) = Interop.mkStyle "height" value
+    /// Sets the width of an element. 
+    ///
+    /// By default, the property defines the width of the content area.
     static member inline width (value: int) = Interop.mkStyle "width" value
+    /// Sets the width of an element. 
+    ///
+    /// By default, the property defines the width of the content area.
     static member inline width (value: ICssUnit) = Interop.mkStyle "width" value
+    /// Sets the size of the element's background image. 
+    ///
+    /// The image can be left to its natural size, stretched, or constrained to fit the available space.
     static member inline backgroundSize (value: string) = Interop.mkStyle "backgroundSize" value
+    /// Sets one or more background images on an element.
     static member inline backgroundImage (value: string) = Interop.mkStyle "backgroundImage" value
     static member inline backgroundImageUrl (value: string) = Interop.mkStyle "backgroundImage" ("url(" + value + ")")
```

Comment:
Missing docs here, maybe "Short-hand for `style.backgroundImage(sprintf "url(%s)" value)` to set the background image using a url".